tests/envs/test_playroom.py

```diff
@@ -0,0 +1,297 @@
+"""Test cases for the boring room vs. playroom environment.
+"""
+
+import numpy as np
+from predicators.src.envs import PlayroomEnv
+from predicators.src import utils
+from predicators.src.structs import Action
+
+
```

Comment:
absolutely beautiful tests! I'll suggest one more. see here: https://github.com/Learning-and-Intelligent-Systems/predicators/blob/master/tests/envs/test_cover.py#L170 the idea is to write down a sequence of actions that you think will do all the interesting things in the task. what I like about this is the `make_video` stuff at the end; so you can temporarily set `make_video = True`, run `pytest -s tests/envs/test_playroom.py`, and it should make a video that plays through your actions. The main thing here is to sanity check the video.